cvmfs/receiver/reactor.h

```diff
@@ -35,7 +35,8 @@ class Reactor {
     kCheckToken,
     kSubmitPayload,
     kCommit,
-    kError
+    kError,
+    kTestCrash  // use to test the gateway
```

Comment:
Let's be more specific: to test what exactly in the gateway

Reply:
Here we are testing what happens when the receiver throw an exception (hopefully there are not false assert in the codepaths).

Before this PR (and related PR) if the receiver crash the gateway just hang waiting.